src/main/java/ladder/domain/resource/direction/Direction.java

```diff
@@ -0,0 +1,18 @@
+package ladder.domain.resource.direction;
+
+import java.util.Random;
+
+public enum Direction {
+
+    RIGHT,
+    LEFT,
+    NEUTRAL;
```

Comment:
방향이라는 enum 을 만든김에 그... 방향을 맞춰보면 어떨까요? :)
```suggestion
public enum Direction {

    LEFT,
    NEUTRAL,
    RIGHT
```